lwt/jbuild

```diff
@@ -1,12 +1,39 @@
+(* -*- tuareg -*- *)
```

Comment:
This `lwt/` library isn't currently tested, but we should test it. So I think it's a good thing to add bisect to this too.